easybuild/easyconfigs/v/VTune/VTune-2018_update1.eb

```diff
@@ -0,0 +1,15 @@
+name = 'VTune'
+version = '2018_update1'
+
+homepage = 'http://software.intel.com/en-us/intel-vtune-amplifier-xe'
+description = """Intel VTune Amplifier XE is the premier performance profiler for C, C++, C#, Fortran,
+ Assembly and Java."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+sources = ['vtune_amplifier_%(version)s.tar.gz']
```

Comment:
@Quarkmus Please include a SHA256 checksum, you can use `eb --inject-checksums` for this (if your EasyBuild version is sufficiently recent).

Reply:
Thanks for the info, I have added this in update b67a84c